CodeEdit/Features/Documents/Controllers/CodeEditWindowController.swift

```diff
@@ -299,6 +299,10 @@ final class CodeEditWindowController: NSWindowController, NSToolbarDelegate, Obs
             }
         }
     }
+
+    @IBAction func closeCurrentTab(_ sender: Any) {
```

Comment:
If preferred, I can remove the `closeCurrentTab()` function from `TabGroupData.swift` and replace this current function with this code:

```swift
@IBAction func closeCurrentTab(_ sender: Any) {
    guard let workspace, let selected = workspace.tabManager.activeTabGroup.selected else { return }
    workspace.tabManager.activeTabGroup.closeTab(item: selected)
}
```

One possible benefit to having `closeCurrentTab` as a function inside `TabGroupData.swift` is that the closing action can be triggered by other components or logic within the application (not just user interface events).